Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelementryConnectionImpl.java

```diff
@@ -6,18 +6,26 @@ public class TelementryConnectionImpl implements TelementryConnection {
 
     private boolean onlineStatus;
 
-    TelementryConnectionEventSimulatorImpl telementryConnectionEventSimulator  = new TelementryConnectionEventSimulatorImpl(42);
+    private final TelementryConnectionEventSimulatorImpl telementryConnectionEventSimulator;
```

Comment:
let the file type to be TelementryConnectionEventSimulator 
Its makes the class loosely coupled